docs/docs/kds-configuration.md

```diff
@@ -207,32 +154,95 @@ kds {
 }
 ```
 
-### Any enclave for a particular product signed using the same key
-This example allows any enclave running any code to access the key as long as it
-is signed with the same signing key, is part of the same product and is running
-on a secure platform.
-
-The combination of a signing key and a product ID allows for unique keys to be
-generated per product, but shared with all versions of enclaves within a single
-product.
+This avoids the denial of service that happens in the more restrictive case above by giving the enclave host time to patch their systems to bring them back to the `SECURE` state.
+The operational implications of enclave updates are not affected however, as access to the derived key still requires that the enclave have a specific code hash.
 
-This makes it easy to release new versions of enclaves, or allows multiple
-different services to share access to persisted data. However, the owner of the
-code signing key must be trusted enclave clients because they have the ability
-to author a new enclave that is able to perform any action on data protected
-using the signing key.
-
-This example is probably most suitable where the service author is trusted, or
-when using an external auditor to validate enclave code and sign the enclave on
-the developer's behalf.
+#### Any enclave for a particular product signed using the same key
+This is an example of a permissive policy constraint, whereby access to the derived key is permitted for enclaves with a matching product ID and signing key.
```

Comment:
In this permissive policy constraint example, access to the derived key is permitted for enclaves with a matching product ID and signing key.